migrations/000007_rename_recommendation_sets_table.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE recommendation_sets RENAME TO historical_recommendation_sets;
```

Comment:
Instead of altering the table, we can make this transition smooth for end users by creating table  as recommendation_sets and keep last reported records in recommendation_sets?

Reply:
recommendation_sets table is already created and it has multiple recommendation for same workload so it is hard to create new historical_recommendation_sets table and move the older recommendation to it. Instead I thought to just rename the recommendation_sets to historical_recommendation_sets which contains the multiple recommendations and a new recommendation_sets is created were only latest 1 recommendation per workload will be saved.